src/tests/DomainHost.t.sol

```diff
@@ -515,11 +551,12 @@ contract DomainHostTest is DssTest {
         // Simulate user getting some gems for this ilk (normally handled by end)
```

Comment:
The `tell()` above this line comment is incorrect as we've switched to `debt` instead of it being a `cure` value.

Reply:
Changed it, let me know if you think is fine now as I didn't understand much the prev one.

Reply:
Yes looks good. Previous one was sending the `cure` value which is how much to discount the debt instead of the absolute value of the debt so previously `tell(X)` would give a final real debt of `grain - X` and now `tell(X)` gives the real debt directly instead of subtracting from the `grain`.

Reply:
Yes yes, I understand what we changed, but the comment itself was a bit confusing for me, so that's why I directly rewrote it differently.